ogame/__init__.py

```diff
@@ -311,6 +311,41 @@ class Celestial:
 
         return Celestial
 
+    def celestial_queue(self, id):
+        response = self.session.get(
+            url=self.index_php + 'page=ingame&component=overview',
+            params={'cp': id}
+        ).text
+        research_time = re.search(r'var restTimeresearch = ([0-9]+)', response)
+        if research_time is None:
+            research_time = datetime.fromtimestamp(0)
+        else:
+            research_time = int(research_time.group(1))
+            research_time = datetime.fromtimestamp(research_time)
+        build_time = re.search(r'var restTimebuilding = ([0-9]+)', response)
+        if build_time is None:
+            build_time = datetime.fromtimestamp(0)
+        else:
+            build_time = int(build_time.group(1))
+            build_time = datetime.fromtimestamp(build_time)
+        shipyard_time = re.search(r'var restTimeship2 = ([0-9]+)', response)
+        if shipyard_time is None:
```

Comment:
is None i think is redundant if shipyard: should be sufficiant
but bedder you return a 0 timestamp. not crashing bots from the enduser